test/Snapshot/ArraySnapshotTest.php

```diff
@@ -42,7 +41,9 @@ public function providerCompare()
      */
     public function testDeepConstructor($value)
     {
-        new ArraySnapshot(['foo' => $value]);
+        $arraySnapshot = new ArraySnapshot(['foo' => $value]);
+
+        self::assertSame(['foo' => $value], $arraySnapshot->getRawData());
```

Comment:
IIRC, this test was just to ensure the constructor didn't throw anything, and it looks useless ot test then the raw data.

But I guess PHPUnit was reporting this test as incomplete or something ? Wasn't there a method or a call that tells phpunit to ignore such errors ?

Reply:
IMO test should perform assertion(s) (or expect exception(s)). Otherwise they are pointless, right?

Reply:
And yes, you're right, tests without assertions (or expected exceptions) are marked as risky, that's why I modified this one.